targets/TARGET_STM/TARGET_STM32F7/device/stm32f7xx_hal_pcd.c

```diff
@@ -300,10 +302,10 @@ __weak void HAL_PCD_MspDeInit(PCD_HandleTypeDef *hpcd)
   */
 HAL_StatusTypeDef HAL_PCD_Start(PCD_HandleTypeDef *hpcd)
 {
-  __HAL_LOCK(hpcd);
```

Comment:
Why was the lock removed?

Reply:
It has been removed in the F4 (and L4 I think too) so I just did the same. Do you think it can cause a problem ?

Reply:
I don't see it being removed in the F4. Can you link to the PR which did this? If there isn't a good reason for removing this I would recommend leaving the lock in.

Reply:
I will double-check the PCD Init,Start, Stop functions between the F2, F4, F7 and L4 devices and align them. I will send this change in a separate PR.